source/help/getting-started/signing-in.rst

```diff
@@ -121,6 +121,16 @@ How Mattermost determines your status
     "**Web Browser**", "You're interacting with Mattermost in a browser", "You're inactive in the Mattermost browser window for 5 minutes", "You close the Mattermost browser window"
     "**Mobile App**", "Mattermost is open", "Mattermost is open with 5 minutes of inactivity", "You switch apps, close Mattermost, or lock your screen"
 
+Desktop v4.4+ bases activity on any user interaction on the computer, so if you are typing or using your mouse, your status will remain online. Your status will be set to away only after 5 minutes of inactivity.
```

Comment:
Thanks for the PR. This info should probably be updated directly in the table rather than below it